standard-components/pom.xml

```diff
@@ -0,0 +1,113 @@
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eobjects.datacleaner</groupId>
+		<artifactId>DataCleaner</artifactId>
+		<version>4.1-SNAPSHOT</version>
+	</parent>
+	<artifactId>DataCleaner-standard-components</artifactId>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.version>4.1-SNAPSHOT</project.version>
```

Comment:
This isn't needed, actually should not be here because it overrides the real property called "project.version" (which is dynamic - provided by Maven itself).